README.md

```diff
@@ -1,43 +1,51 @@
+### Voting Members
 
-| Vocabulary | Definition |
-|---|---|
-| Bootstrap | This repo, team, and process. |
-| Contributor | You, and anyone that interacts with this repo. |
-| Ignition | The decision to move forward with the new governance structure. |
-| TLCs | The Top Level Councils of the project – Currently proposed as CPC and C3 (defined below) |
-| CPC | A new proposed Top Level Committee – The "Cross Project Council |
-| C3 | A new proposed Top Level Committee – The "Cross Community Council |
-| CNCF | The Cloud Native Computing Foundation (Prior Art) |
-| COC | Code of Conduct |
-| MVF | Minimum Viable Foundation – The bare minimum governance documentation to operate a function joint foundation |
-| Proposal | The staged proposal feedback and iteration process followed in the `/proposals` directory. |
+#### Impact Project Representatives
 
-## Deliverables
-The deliverables from this body will be:
+Each Impact Project may appoint 2 representatives to the CPC as outlined in the [CPC Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#voting-members)
 
-  - [ ] A recommendations document to help inform the new Foundation's by-laws.
-  - [ ] A recommendations document to help inform the TLCs' charters.
-  - [ ] A collection of fully drafted governance documents required for "Ignition". This is "Minimum Viable Foundation" (MVF).
-  - [ ] A collection of program, initiative, and governance recommendation documents not required for MVF. After Ignition these will be passed to the appropriate bodies (CPC, C3, etc.) to execute.
+- **Appium**: Dan Graham (@dpgraham) & Jonah Stiennon(@Jonahss)
+- **Dojo**: Matt Gadd (@matt-gadd) & Dylan Schiemann (@dylans)
+- **jQuery**: Dave Methvin (@dmethvin) & Timmy Willison (@timmywil)
+- **Node.js**: Matteo Collina (@mcollina ) & Joe Sepi (@joesepi)
+- **webpack**: Sean Larkin (@TheLarkInn) & Even Stensberg (@evenstensberg)
 
+#### Growth & At Large Project Representatives
 
-## Making proposals
+According to the [CPC Charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#voting-members), each of the Growth and At Large Projects may nominate a candidate to fill one of two voting seats on the CPC which represent this group of projects as a whole. Currently, the four individuals listed below were nominated during the bootstrap process and [the bootstrap committee decided](https://github.com/openjs-foundation/cross-project-council/issues/152) that all nominees would be given a voting seat until a process for nomination and election of these representatives was formalized.
+
+- **Architect**: Kris Borchers (@kborchers)
+- **messageformat**: Eemeli Aro (@eemeli)
+- **Mocha**: Christopher Hiller (@boneskull)
+- **webhint**: Antón Molleda (@molant)
+
+### Regular Members
```

Comment:
To be filled in ?

Reply:
@evenstensberg the plan is to  have people PR themselves in as regular members after this lands.

Reply:
Should be empty for now 👍

Reply:
I'm going to comment this out until we have members to add